.gitignore

```diff
@@ -16,5 +16,10 @@ crawlers/ecolex/!treaty decisions/.*
 crawlers/ecolex/literature/*
 crawlers/ecolex/!literature/.*
 
+crawlers/eurlex/celex_nums/*
+crawlers/eurlex!celex_nums/.*
```

Comment:
Is the exclamation character intended to be there?
Should it be `crawlers/eurlex/!celex_nums/.*`?

Reply:
Git does not automatically add folders if there arent any files in them, therefore if you want to push  some folder structure to the repository, you add a `.gitkeep` file to each of the folders.. The second line above then ignores everything but the files starting with a dot.